src/components/HeroImage/styles.scss

```diff
@@ -0,0 +1,71 @@
+// Mobile portrait
+.hero {
+	position: fixed;
+	width: 100%;
+	height: 100%;
+	top: 0;
+	right: 0;
+	z-index: 0;
+}
+
+	.glow {
+		position: absolute;
+		width: 1em;
+		height: 1em;
+		top: 0;
+		left: 50%;
+		font-size: 100vw;
+
+	}
+		.glow::before {
+			content: "";
+			display: block;
+			width: 100%;
+			height: 100%;
+			background-color: yellow;
+			border-radius: 100%;
+			transform: translate(-50%, -50%);
+			filter: blur(0.2em);
+		}
+
+	.image {
+		display: block;
+		width: 100%;
+		height: 100%;
+		background-color: crimson;
+	}
+
+// Mobile landscape
+@media (orientation: landscape) {
+	.hero {
+		width: 75%; // GUESS
+	}
+
+		.glow {
+			font-size: 50vw; // GUESS
+		}
+
+			.glow::before {
+				transform: translate(-25%, -66%); // GUESS
+			}
+}
+
+// Tablet portrait
+@media (min-width: 768px) {
+
+}
+
+// Tablet landscape
+@media (min-width: 768px) and (orientation: landscape) {
+
+}
+
+// Desktop portrait
+@media (min-width: 1200px) and (max-aspect-ratio: 8/9) {
+
+}
+
+// Desktop landscape
+@media (min-width: 1200px) {
```

Comment:
let's clean-up these unused media queries and just write'em as we need them from here on out 👌